nxfilter.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "nxfilter",
+  "release": "12.2-RELEASE",
+  "artifact": "https://github.com/xTITUSMAXIMUSX/iocage-plugin-nxfilter.git",
+  "properties": {
+        "allow_raw_sockets":"1",
+        "boot":"on"
+          },
+  "pkgs": [
+        "curl",
+        "openjdk8-jre"
+        ],
+  "packagesite": "http://pkg.FreeBSD.org/${ABI}/quarterly",
```

Comment:
Are you using `quarterly` for some special reason, or isn't `latest` better in most cases?

Reply:
Yes. Latest would be best. I have that updated. Thanks for pointing that out.